internal/venice-common/src/main/java/com/linkedin/venice/writer/VeniceWriter.java

```diff
@@ -190,9 +190,9 @@ public class VeniceWriter<K, V, U> extends AbstractVeniceWriter<K, V, U> {
    */
   public static final int VENICE_DEFAULT_VALUE_SCHEMA_ID = -1;
 
-  private static final byte[] EMPTY_BYTE_ARRAY = new byte[0];
```

Comment:
Not protected?

Reply:
I am using it in a separate unit test and since it is `static final`, I guess it is fine.

Reply:
you may use `EMPTY_BYTE_ARRAY` from io commons.